src/pages/VenueLandingPage/VenueLandingPageContent.tsx

```diff
@@ -60,7 +63,7 @@ const VenueLandingPageContent: React.FC<VenueLandingPageContentProps> = ({
   const onJoinClick = () => {
     if (!venueId) return;
 
-    const venueEntrance = venue.entrance && venue.entrance.length;
+    const venueEntrance = world?.entrance?.length;
     window.location.href =
       user && !venueEntrance
```

Comment:
Might as well inline the variable, or rename to `hasEntrance`...
```suggestion
      user && !world?.entrance?.length
```

Reply:
Fixed in https://github.com/sparkletown/sparkle/pull/2546/commits/08a3290e454507ee3163e87d8f4078506f8d7b8f